packages/v4/patternfly-docs/content/training/html-css/fundamentals.md

```diff
@@ -0,0 +1,182 @@
+PatternFly consists of isolated and modular structures that fall into three categories:
+- Components are modular and independent structures concerned with presentation
+- Layouts allow for organizing and grouping their immediate children on the page
+- Demos illustrate how to assemble complex structures with components and layouts only
+
+## Components
+In PatternFly, components which cannot be broken down into smaller parts are known as the basic building blocks of user interfaces. Examples include the button, label and badge components. A codesandbox link has been provided to experiment with and follow the proceeding steps.
+
+<Card style={{maxWidth: "400px"}} component="div" isSelectableRaised onClick={() => window.open("https://codesandbox.io/s/html-fundamentals-start-hy2erg?file=/index.html", '_blank')}>
+  <CardBody>Codesandbox - HTML fundamentals</CardBody>
+</Card>
+</PageSection>
+<PageSection>
+### Step 1. Add a button component.
+In the `<body>` of the index.html file, add a button element, and apply the PatternFly styles to it so that it is a PatternFly button by adding the class `pf-c-button`. Apply the ‘primary’ styles to the button to make it a bright blue color by adding the `pf-m-primary` class to the button as well.
+
+<CopyCodeBlock>
+{`<button class="pf-c-button pf-m-primary" type="button">\n  Primary\n</button>`}
+</CopyCodeBlock>
+
+**Note:** A simple component can be reused multiple times.
+
+<CopyCodeBlock>
+{`<button class="pf-c-button pf-m-primary" type="button">
+Primary\n</button>\n<button class="pf-c-button pf-m-primary" type="button">
+Primary\n</button>`}
+</CopyCodeBlock>
+
+### Step 2. Build more complex components
+Simple components can be combined to make more complex components that are still reusable.
+When simpler components are used within more complex components, the layout of the elements are defined in the stylesheet for the complex component.
+In this step, add a badge component to the chip component.
+
+#### Step 2.1 
+To build a chip component replace the contents of the `<body>` in the index.html file with the following code snippet.
+
+<CopyCodeBlock>
+{`<div class="pf-c-chip">
+<span class="pf-c-chip__text">
+  Chip
+</span>
+<button class="pf-c-button pf-m-plain">
+  <i class="fas fa-times"></i>
+</button>\n</div>`}
+</CopyCodeBlock>
+
+This is the default chip component that already has another component in it: the button. This is why the chip can be considered as a more complex component.
+
+#### Step 2.2
+Add the badge inside the chip. To do this, add this block of code between pf-c-chip__text and pf-c-button.
+
+<CopyCodeBlock>
+{`<span class="pf-c-badge pf-m-read">
+7\n</span>`}
+</CopyCodeBlock>
+
+**Note:** Remember this is a component and not a demo because the chip component includes styles that handle how the badge looks within the chip. For example, when the badge component is added to the chip, it receives styling that gives it a margin. You can read more in the documentation here
```

Comment:
```suggestion
**Note:** Remember, this is a component and not a demo because the chip component includes styles that handle how the badge looks within the chip. For example, when the badge component is added to the chip, it receives styling that gives it a margin. 
```

Reply:
We can add back a more specific documentation link. I'm not sure what this is intended to point to?